website/versions/source-facebookmarketing.json

```diff
@@ -0,0 +1 @@
+{ "latest": "plugins-source-facebookmarketing-v1.0.0" }
```

Comment:
We don't need this file. The Website will show the plugin as "Unpublished" until we release it, and then the file will be created automatically.